book/website/community-handbook/contributing.md

```diff
@@ -108,33 +104,65 @@ You can also read the [Authoring section of the MyST Markdown Guide](xref:myst-g
 
 1. Create a feature branch on your fork
 2. Commit changes
 3. Open a pull request to _The Turing Way_'s repository
   - Open a draft pull request if the work is still in progress
-  - Open a pull request if you feel the work is ready for review
+  - Open a pull request if you feel the work is ready for {term}`review`
 4. Iterate on changes in response to CI and reviewer comments
-5. Merge when all issues are addressed and changes are approved
+5. Merge when all {term}`issues` are addressed and changes are approved
 
 ### Best Practice
 
 Here are some tips for best practice working with git:
 
 - [Keep your fork up to date](https://help.github.com/articles/syncing-a-fork) with the main repository
 - Try to keep changes focused.
-  If you submit many changes at once it will be much more difficult to review.
-- Make small commits often.
-- Write [good commit messages](https://chris.beams.io/git-commit).
+  If you submit many changes at once it will be much more difficult to {term}`review`.
+- Make small {term}`commits <commit>` often.
+- Write [good {term}`commit` messages](https://chris.beams.io/git-commit).
 - Do not re-write history!
-- Open draft pull requests soon, to get early feedback.
+- Open draft {term}`pull requests <pull request>` soon, to get early feedback.
+
+(ch-contributing-low-effort)=
+### Low Effort or Engagement
+
+An important part of our community is putting in effort to mentoring contributors, particularly those new to open source.
+However, we also need to protect our maintainers' time so they do not get burned out reviewing poor quality contributions or trying to mentor unengaged contributors.
+And so, pull requests may be closed if we feel they are unlikely to result in a _tangible improvement_ to the book, or the maintainer investment required would not match the value of the contribution.
+
+:::{important}
+There is no size threshold contributions to be accepted.
+For example, single typo fixes are great contributions.
+:::
+
+Features that mark a contribution as low effort include:
+
+- Mostly or entirely superficial changes, such as whitespace changes or replacing words with synonyms.
+- Ignoring book structure and the [style guide](#ch-style), such as not updating the table of contents, or ignoring directory structure.
+- Apparently machine-generated contributions with little or no human effort to review and edit.
+  This may be apparent in comments as:
+    - Irrelevant, unclear text.
+    - Disconnection between {term}`pull request` comments and changes.
+    - Hallucinated references.
+  And in code/contributions as:
+    - Verbose and vague text.
+    - Long sections of plain text lacking, for example, cross-references, citations, figures, admonitions.
+    - Hallucinated references.
+
+Features that mark a contribution as low engagement include:
+
+- Not responding to reviewers.
+- Ignoring reviewers and instead making unrelated changes.
+
```

Comment:
```suggestion
- Not using project templates for {term}`issues` or pull requests.
```

Reply:
This might be a bit more contentious, I often skip the templates if I think I know what to write or the changes are clear (or I'm feeling lazy 😅)

Reply:
That works! I've just noticed that most AI stuff comes in without a proper PR template completed which is a dead giveaway - but I agree we don't want to be super rigid if people have made an effort...